src/components/users/ForgotPassword.js

```diff
@@ -78,13 +85,15 @@ const ForgotPassword = ({ setForgotPassword }) => {
               inputName="email"
               inputValue={email}
               inputOnChange={handleChange}
-              error={isSubmitted && !email}
+              error={'email' in errors}
```

Comment:
don't you have these names defined somewhere?

Reply:
What do you mean?